.classpath

```diff
@@ -18,6 +18,7 @@
 	<classpathentry excluding="**/.svn/**|.svn/" kind="src" path="modules/core/petra/petra-sql-dsl-api/src/main/java"/>
 	<classpathentry excluding="**/.svn/**|.svn/" kind="src" path="modules/core/petra/petra-sql-dsl-spi/src/main/java"/>
 	<classpathentry excluding="**/.svn/**|.svn/" kind="src" path="modules/core/petra/petra-string/src/main/java"/>
+	<classpathentry excluding="**/.svn/**|.svn/" kind="src" path="modules/core/petra/petra-pattern-mapper/src/main/java"/>
```

Comment:
hey @arthurchan35, I believe there is a typo here, isn't there?

Reply:
@csierra ,

Thanks Carlos, I looked at them after commiting, but still couldnt see this one..

Reply:
sorry to be the bearer of bad news... :sweat: